presto-orc/src/main/java/com/facebook/presto/orc/reader/LongStreamReader.java

```diff
@@ -90,4 +91,11 @@ public String toString()
                 .addValue(streamDescriptor)
                 .toString();
     }
+
+    @Override
+    public void close()
+    {
+        directReader.close();
```

Comment:
Currently we know that `close` doesn't throw, but just to make the code future proof we can use guava `Closer` here to make sure that both readers get closed.